.travis.yml

```diff
@@ -16,12 +16,12 @@ before_install:
 
 install:
   - export GIT_FULL_HASH=`git rev-parse HEAD`
-  - conda create --yes -n testenv python=$TRAVIS_PYTHON_VERSION
+  - conda create python=$TRAVIS_PYTHON_VERSION
   - source activate testenv
-  - conda install github3.py 
 
 script:
-  - git clone https://github.com/CJ-Wright/cf-graph.git
-  - pip install git+git://github.com/drdoctr/doctr.git
-  - python find-feedstocks.py
-  - doctr deploy . --built-docs ./cf-graph/;
+  - git clone https://github.com/regro/cf-graph.git
+  - cd cf-graph
+  - conda install --file requirements/run
+  - python 00-find-feedstocks.py
```

Comment:
I noticed that the name of this script is inconsistent with the others. The other scripts are 01-make_grapy.py, 02-graph_upstream.py, and 03-auto_tick.xsh, but this script is 00-find-feedstocks.py with a hyphen instead of an underscore.

Reply:
Good point! I'll change the filename over in the main repo.